clinica/utils/freesurfer.py

```diff
@@ -46,6 +46,7 @@ def generate_regional_measures_alt(
         stats_folder = os.path.join(
             os.path.expanduser(segmentation_path), subject_id, "stats"
         )
+        print("ZE STAT FOLDER: ", stats_folder)
```

Comment:
I love that print statement :-)

Reply:
mm though call.. but I'd be inclined to remove it!

Reply:
So do I ! Better cut it out, though :)